tests/integration_tests/test_retrievers.py

```
@@ -0,0 +1,25 @@
+from typing import Type
```

Ruff (0.8.2)

`typing.Type` is deprecated, use `type` instead

(UP035)